src/mono/wasm/build/WasmApp.targets

```diff
@@ -151,6 +170,9 @@
 
       <WasmNativeAsset Include="$(MicrosoftNetCoreAppRuntimePackRidNativeDir)$(WasmIcuDataFileName)" Condition="'$(InvariantGlobalization)' != 'true'" />
       <WasmNativeAsset Include="$(MicrosoftNetCoreAppRuntimePackRidNativeDir)dotnet.timezones.blat" />
+
+      <WasmExtraFilesToDeploy Include="$(TargetDir)publish\runtimeconfig.bin" TargetPath="managed" />
+      <WasmFilesToIncludeInFileSystem Include="$(TargetDir)publish\runtimeconfig.bin" />
```

Comment:
Yes. The first one is to copy `runtimeconfig.bin` file to `AppBundle` folder. The second one is to make sure the file is visible in Wasm system.

Reply:
> The first one is to copy runtimeconfig.bin file to AppBundle folder. 

Why is this needed? Including it in `WasmFilesToIncludeInFileSystem` should ensure that it will be copied to the app bundle.

Reply:
Then, line 174 is probably not needed. I added line 174 before 175.